client/src/app/tabs/form/__tests__/FormEditorSpec.js

```diff
@@ -266,6 +267,26 @@ describe('<FormEditor>', function() {
         expect(onChangedSpy).to.have.been.calledOnce;
       });
 
+
+      it('should provide selection entries', async function() {
+
+        // given
+        const onChangedSpy = (state) => {
+
+          const editMenuEntries = getSelectionEntries(state);
+
+          // then
+          expect(state.editMenu).to.deep.include(editMenuEntries);
+        };
+
+        const { instance } = await renderEditor(schema, {
+          onChanged: onChangedSpy
+        });
+
+        // when
+        instance.handleChanged();

```

Comment:
Hmm so the test ends after this call. If `handleChanged` ever gets async, failure in the `onChangeSpy` will only be revealed as an unhandled promise rejection -> ignored by the test framework.

Reply:
You're right. All of these tests are probably broken. 🤡 Will fix in a separate PR.